packages/form-core/src/FormApi.ts

```diff
@@ -207,6 +211,9 @@ export class FormApi<
 
           const isTouched = fieldMetaValues.some((field) => field?.isTouched)
 
+          const isDirty = fieldMetaValues.some((field) => field?.isDirty)
+          const isPristine = !isDirty
```

Comment:
Can we do this same trick inside of `FieldApi.ts` to make `isPristine` a non-actual meta field? (actually asking, IDK off the top)

Reply:
Ok, I think now we should be using it: see [`packages/form-core/src/FieldApi` line 325](https://github.com/TanStack/form/blob/314db2335ddda223d48e81a7865363cb652b8473/packages/form-core/src/FieldApi.ts#L325).